tests/Microsoft.Identity.Test.Unit.net45/CacheV2Tests/FileIOTests.cs

```diff
@@ -42,6 +42,7 @@
 namespace Microsoft.Identity.Test.Unit.CacheV2Tests
 {
     [TestClass]
+    [Ignore]
```

Comment:
Don't we want to add a [WorkItem] pointing to an issue that needs to be fixed to re-enable the tests?